cpp/src/strings/search/findall.cu

```diff
@@ -105,17 +103,12 @@ std::unique_ptr<column> findall(strings_column_view const& input,
   auto d_prog = regex_device_builder::create_prog_device(prog, stream);
 
   // Create lists offsets column
-  auto offsets   = count_matches(*d_strings, *d_prog, strings_count + 1, stream, mr);
-  auto d_offsets = offsets->mutable_view().data<size_type>();
-
-  // Convert counts into offsets
-  thrust::exclusive_scan(
-    rmm::exec_policy(stream), d_offsets, d_offsets + strings_count + 1, d_offsets);
-
-  // Create indices vector with the total number of groups that will be extracted
-  auto const total_matches =
-    cudf::detail::get_value<size_type>(offsets->view(), strings_count, stream);
+  auto const sizes = count_matches(*d_strings, *d_prog, strings_count, stream, mr);  //+1
```

Comment:
What does the `//+1` mean here? I see previously the code passed `strings_count + 1`. Did the behavior change?

Reply:
Ah, good catch. That was left-over note-to-self. The +1 is not needed since the number of counts equals the strings_count. Previously we were storing the counts in an offsets column temporarily.

Reply:
Removed the comment here: https://github.com/rapidsai/cudf/pull/14745/commits/ba0d432040622e9d99a1f9c8bbaf2fed2575d811